src/Altinn.Platform/Altinn.Platform.Storage/Storage/Altinn.Platform.Storage.csproj

```diff
@@ -22,9 +22,9 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.AzureKeyVault" Version="3.1.14" />
     <PackageReference Include="Microsoft.Extensions.Logging.ApplicationInsights" Version="2.17.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.1.1" />
-    <PackageReference Include="Swashbuckle.AspNetCore.Newtonsoft" Version="6.1.1" />
-    <PackageReference Include="Swashbuckle.AspNetCore.SwaggerUI" Version="6.1.1" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.1.2" />
```

Comment:
As we have weakly patching routines it would be enough to update the single dependency that you need. That way the testing and potential bugs can only be traced back to the new feature, rather than the pathing of a dependency used by another feature in the component.

Reply:
Noted:)